src/main/java/domain/Line.java

```diff
@@ -0,0 +1,56 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static domain.LineState.UNMOVABLE_STATE;
+
+public class Line {
+
+    private final List<Boolean> points = new ArrayList<>();
+
+    public Line(int personCount, RandomGenerator generator) {
+        addPoints(personCount, generator);
+    }
+
+    public List<Boolean> getPoints() {
+        return points;
+    }
+
+    public int getPointsSize() {
+        return points.size();
+    }
+
+    public boolean isMovablePoint(int pointIndex) {
+        return points.get(pointIndex);
+    }
+
+    private void addPoints(int personCount, RandomGenerator generator) {
+        addRandomPoint(generator);
+        for (int pointIndex = 1; pointIndex < personCount - 1; pointIndex++) {
+            addConditionalPoint(generator, pointIndex);
+        }
+    }
+
+    private void addConditionalPoint(RandomGenerator generator, int pointIndex) {
+        if(isSuccessive(pointIndex)){
+            addPoint(UNMOVABLE_STATE.getState());
+        }
+        if(!isSuccessive(pointIndex)){
+            addRandomPoint(generator);
+        }
+    }
+
+    private boolean isSuccessive(int index){
```

Comment:
```suggestion
    private boolean isSuccessive(int index) {
```

> 코드 자동 정렬하기 (Reformat Code)
MacOS: Cmd + Opt + l
Win/Linux: Ctrl + Alt + l

[[IntelliJ] intellij 유용한 단축키 정리](https://gmlwjd9405.github.io/2019/05/21/intellij-shortkey.html) 😉